components/src/molecules/ParametersTable/ParametersTable.stories.tsx

```diff
@@ -1,15 +1,10 @@
-import * as React from 'react'
-import { ParametersTable } from '@opentrons/components'
-import type { Story, Meta } from '@storybook/react'
-import type { RunTimeParameter } from '@opentrons/shared-data'
-
-export default {
-  title: 'Library/Molecules/ParametersTable',
-} as Meta
+import * as React from 'react-remove-scroll'
+import { Flex } from '../../primitives'
+import { SPACING } from '../../ui-style-constants'
+import { ParametersTable } from '.'
```

Comment:
thanks for fixing the circular dependency before! but nit: maybe 
```suggestion
import { ParametersTable } from './index'
```